docs/guides/external-mgs/readme.md

```diff
@@ -0,0 +1,151 @@
+---
+authors: Matt Richerson <matthew.richerson@hpe.com>
+categories: provisioning
+---
+
+# Lustre External MGS
+
+## Background
+
+Lustre has a limitation where only a single MGT can be mounted on a node at a single time. In the NNF cluster, this means the total number of Lustre file systems is limited to the number of NNF nodes. It also places extra scheduling constraints on the WLM to ensure that the MGTs don't conflict.
+
+A Lustre MGS is able to provide management for multiple separate Lustre file systems. The NNF software can be configured to take advantage of this to extend the maximum number of Lustre file systems beyond the number of NNF nodes. To do this, one or more Lustre MGSs must be specified that have a lifetime longer than any of the Lustre file systems that are using it. This can be done in a few different ways hrough the NNF software:
```

Comment:
```suggestion
A Lustre MGS provides management for multiple, separate, Lustre file systems. The NNF software can be configured to take advantage of this to extend the maximum number of Lustre file systems beyond the number of NNF nodes. To do this, one or more Lustre MGSs must be specified that have a lifetime longer than any of the Lustre file systems that are using it. This can be done in a few different ways through the NNF software:
```